include/model/statedata.h

```diff
@@ -48,7 +48,7 @@
 #define ARCH_NODE_STATE_ON_CORE(_state, _core) _state
 #define NODE_STATE_ON_CORE(_state, _core)      _state
 
-#define CURRENT_CPU_INDEX() 0
```

Comment:
any issues when a plain 0 was used  before?

Reply:
Yes. Note that when the error message is printed there is a `SMP_TERNARY(getCurrentCPUIndex(), 0lu)` there actually, using `0lu` instead of `0`.